frontend/amundsen_application/static/js/features/Widgets/Tags/index.tsx

```diff
@@ -36,9 +36,11 @@ export type TagsListContainerProps = StateFromProps &
 
 export class TagsListContainer extends React.Component<TagsListContainerProps> {
   componentDidMount() {
-    this.props.getAllTags();
+    getAllTags();
   }
 
+  getAllTags = () => this.props;
+
```

Comment:
Is this working?
I would actually expect this to become:
```
 componentDidMount() {
    const {getAllTags} = this.props;
     
    getAllTags();
  }
```

Reply:
I am a bit puzzled. Shouldn't it be this.getAllTags()?

Reply:
I'm not 100% sure it actually worked how I had it, because the tags don't populate when I run it on localhost (it was that way on the main branch, even before I made any changes). I now have it the way you mentioned:

```
  componentDidMount() {
    const { getAllTags } = this.props;

    getAllTags();
  }
```